### PR TITLE
feat: add channel management

### PR DESCRIPTION
**Pull request by hejkerooo**
- [x] Channel management CRUD
- [x] Swagger documentation
- [x] Tests - I need to setup "basic" steps (setting up private key, mocking SSI-HUB etc.)

### CODE REVIEW

File: apps/dsb-client-gateway-api/src/app/modules/channel/service/channel.service.ts [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+@Injectable()
+export class ChannelService {
+  protected readonly logger = new Logger(ChannelService.name);
+
+  constructor(
+    protected readonly channelRepository: ChannelRepository,
+    protected readonly dsbApiService: DsbApiService
+  ) {}
+
+  public async createChannel(payload: CreateChannelDto): Promise<void> {
```
Comment (vikaskumr) [inline apps/dsb-client-gateway-api/src/app/modules/channel/service/channel.service.ts:21]: should we not return any message or id when channel is successfully created

File: apps/dsb-client-gateway-api/src/app/modules/dsb-client/service/dsb-api.service.ts [hunk 2]
```diff
@@ -133,7 +155,7 @@ export class DsbApiService implements OnModuleInit {
               amount,
             },
             headers: {
-              Authorization: `Bearer ${this.authToken}`,
+              Authorization: `Bearer ${this.didAuthService.getToken()}`,
             },
           })
         ).catch((err) => this.handleRequestWithRetry(err, retry));
```
Comment (vikaskumr) [inline apps/dsb-client-gateway-api/src/app/modules/dsb-client/service/dsb-api.service.ts:161]: this function ```handleRequestWithRetry``` does not handle error codes returned from message broker API like 
it returns to code 51 when a duplicate topic name exists.

File: apps/dsb-client-gateway-api/src/app/modules/channel/service/channel.service.ts (hunk 1 above)
Comment (hejkerooo) [reply apps/dsb-client-gateway-api/src/app/modules/channel/service/channel.service.ts:21]: we are returning `201` status code, channel id === channel name

File: apps/dsb-client-gateway-api/src/app/modules/dsb-client/service/dsb-api.service.ts [hunk 3]
```diff
@@ -87,6 +87,28 @@ export class DsbApiService implements OnModuleInit {
     }
   }
 
+  public async getTopicsByOwnerAndName(
+    name: string,
+    owner: string
```
Comment (vikaskumr) [inline apps/dsb-client-gateway-api/src/app/modules/dsb-client/service/dsb-api.service.ts:92]: does it mean here ownerDid and applicationNamespace?

**Review by vikaskumr — APPROVED** [review]
Looks good to me except I have also made some topics API so we will have to sort it out in one function to avoid duplicate functions and error handling function in dsb-service

**Pull request merged**